db/migrate/20160324174132_create_container_deployments.rb

```diff
@@ -0,0 +1,18 @@
+class CreateContainerDeployments < ActiveRecord::Migration[5.0]
+  def change
+    create_table :container_deployments do |t|
+      t.string     :kind
+      t.string     :version
+      t.boolean    :containerized
+      t.string     :method
```

Comment:
`method` is a bad column name as it conflicts with [Object#method](http://ruby-doc.org/core-2.2.4/Object.html#method-i-method)